xml/docu_styleguide.structure.xml

```diff
@@ -1317,14 +1438,14 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
           <para>
```

Comment:
In regards to screenshot, I think there is one item missing. Sometimes I see that screenshots are (ab)used for code blocks. This is wrong.

Screenshots that just shows code or other text as the main purpose should _always_ use `<screen>` or `[source]` markup.

Maybe you could add that?